app/webpacker/src/stylesheets/_search.scss

```diff
@@ -32,3 +32,9 @@
     text-decoration: underline;
   }
 }
+
+div.facet-classifications-tag-container {
+  @media (max-width: $mobile-max-width) {
```

Comment:
Should this be using the govuk responsive mixins?

Reply:
yes, I've replaced it with `govuk-media-query($until: tablet)`,
I've tested it and works well 👍